src/test/java/org/folio/rest/impl/TransactionTest.java

```diff
@@ -44,7 +43,7 @@ public class TransactionTest extends TestBase {
 
   private static final String FY_FUND_QUERY = "?query=fiscalYearId==%s AND fundId==%s";
   public static final String ALLOCATION_SAMPLE = "data/transactions/zallocation_AFRICAHIST-FY22_ANZHIST-FY22.json";
-  public static final String TRANSFER_NOT_ENOUGH_MONEY_ERROR_TEXT = "Transfer was not successful. There is not enough money Available in the budget to complete this Transfer";
+//  public static final String TRANSFER_NOT_ENOUGH_MONEY_ERROR_TEXT = "Transfer was not successful. There is not enough money Available in the budget to complete this Transfer";
```

Comment:
If this code not used any more - please delete it

Reply:
ok, I will delete